cli/integration_tests/sync_test.go

```diff
@@ -0,0 +1,38 @@
+//go:build integration
+
+package integration_tests
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"path"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func addDestination(t *testing.T, configFile string) {
```

Comment:
We should be able to remove this once https://github.com/cloudquery/cloudquery/pull/1847 is merged